### PR TITLE
Added new topic

### PR DESCRIPTION
**Pull request by drazenkljajic**
This new topic is just a starting point for implementing this new idea.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ test
 Some description.
 
 Another description.
+
+New topic description.
```
Comment (dkljajic) [inline README.md]: Please provide more details about the new topic.

**Comment by dkljajic** [comment]
If you make that change, I'll merge this.

**Comment by drazenkljajic** [comment]
I improved the description as requested.

**Comment by dkljajic** [comment]
OK. This looks fine.

**Comment by drazenkljajic** [comment]
Some **comments**:

- [X] Write the code
- [ ] Write all the tests
- [ ] Document the code

**Pull request merged**